src/components/routes/Host/HostListingCard/HostListingCard.tsx

```diff
@@ -62,11 +62,14 @@ const HostListingCard = (props: Props): JSX.Element => {
           </BeeLink>
           <Button background="core" color="white" size="small" onClick={() => deleteListing(id)}>
             Delete
-          </Button>          
+          </Button>
+          <div className='host-listing-publish'>
           <label htmlFor={`publish-${id}`} title={canPublish ? '' : INCOMPLETE_LISTING}>
             <span className={canPublish ? '' : 'host-listing-meta--disabled'}>Publish</span>
             <Switch checked={isActive} disabled={!canPublish} onColor={hexColor('correct')} onChange={() => toggleListing(id)} id={`publish-${id}`} />
           </label>
+          {!canPublish && <p className='host-listing-notice'>{INCOMPLETE_LISTING}</p>}
```

Comment:
Whitespace here too; expect `<label ...>` and `{!canPublish...}` to be indented

Reply:
👍  why cant lint find this i wonder